src/common/not_before_queue.h

```diff
@@ -210,11 +210,21 @@ class not_before_queue_t {
   /**
    * advance_time
    *
-   * Advances the eligibility cutoff, argument must be non-decreasing in
-   * successive calls.
+   * Advances the eligibility cutoff.
+   * Argument should be non-decreasing in successive calls.
+   *
+   * As for the scrub queue the cutoff is a time value, and as we have
+   * encountered rare cases of clock resets (even the monotonic clock
+   * can be slightly adjusted backwards on some kernels), "backward"
+   * updates will be tolerated - ignored and not assert.
```

Comment:
Fine. The tracker links https://tracker.ceph.com/projects/ceph/repository/ceph/revisions/8026f1f3d0405db8323ab16b97a74dc5c4b2cbe6.